### DISCUSSION TITLE
feature request: custom log file name

### DISCUSSION THREAD

**Discussion by hdez-es** [DigitalRuby/IPBan#215] [General]
Hello,
Would it be possible to add a new key in the ipban.config to specify the name of the log file and the folder? for example 
\<add key="LogFile" value="c:\logs\ipban.txt"/>
Thanks

**Comment by jjxtra** [comment]
You should be able to edit the nlog.config file to do this. Once done editing, the service must be restarted.

https://nlog-project.org/config/

**Comment by hdez-es** [comment]
Ok. It's done. Thanks
